File: Lista 1 sala/C/atividade03.c
// 3)- Calcular e exibir o valor do volume de uma lata de óleo, usando a fórmula:
// VOLUME <- 3.14159 * R * R * ALTURA.


#include <stdio.h>

int main() {
    float volume, raio, altura, volume_da_lata;

    printf("Digite o volume da lata em litros: ");
    scanf("%f", &volume);
    
    printf("Digite o raio da lata: ");
    scanf("%f", &raio);
    
    printf("Qual a altura da lata? ");
    scanf("%f", &altura);
    
    if (raio <= 0) {
        printf("O valor de raio nao pode ser negativo ou zero\n");
        return 1; 
    }
    
    volume_da_lata = 3.14159 * raio * raio * altura;
    
   
    printf("Este e o volume da lata: %.2f metros cubicos\n", volume_da_lata);
    
    return 0;
}

File: Lista 1 sala/Python/atividade03.py
# 3)- Calcular e exibir o valor do volume de uma lata de óleo, usando a fórmula:
# VOLUME <- 3.14159 * R * R * ALTURA.



volume = float(input("Qual o volume da lata:",))
raio = float(input("Qual o valor do raio da lata em cm?"))
altura = float(input("Qual o valor da altura?"))

if raio <= 0:
    print("O valor de raio não pode ser negativo")


volume_da_lata =  raio * raio * altura

print("seu volume da lata é 30{:.1f}".format (volume_da_lata))
